### PR TITLE
rocAL - Fix C++ tests

### PR DESCRIPTION
**Pull request by SundarRajan98**
- Added rocalSetSeed to C++ tests for generating seed sequence vector used in rocAL parameters

### CODE REVIEW

**Review by LakshmiKumar23 — APPROVED** [review]

File: apps/image_augmentation/image_augmentation.cpp [hunk 1]
```diff
@@ -107,6 +107,7 @@ int main(int argc, const char** argv) {
         std::cout << "Could not create the rocAL contex\n";
         return -1;
     }
+    rocalSetSeed(0);
```
Comment (rrawther) [inline apps/image_augmentation/image_augmentation.cpp:110]: If every application which uses rocAL needs to call this function, I would suggest to add this to the constructor of the mastergraph (pipeline)

**Review by rrawther — CHANGES_REQUESTED** [review]
please address the review comments

File: apps/image_augmentation/image_augmentation.cpp (hunk 1 above)
Comment (SundarRajan28) [reply apps/image_augmentation/image_augmentation.cpp]: Set the seed for the parameter factory instance in Mastergraph constructor. Removed the rocalSetSeed(0) calls from the C++ tests as they are automatically done.

**Comment by LakshmiKumar23** [comment]
@kiritigowda can be merged

**Pull request merged**